friends.html

```diff
@@ -1 +1,185 @@
+          </div>
+        </div>
+        <div class="row outlined">
+          <div class="col-md-9 col-md-offset-3">
+            <span>
+              <button class="btn btn-lg fb-nav-buttons">Timeline</button>
+              <button class="btn btn-lg fb-nav-buttons fb-nav-buttons-selected">About</button>
+              <button class="btn btn-lg fb-nav-buttons">Photos</button>
+              <button class="btn btn-lg fb-nav-buttons">Friends</button>
+              <a class="edit-profile-anchor">Edit Profile</a>
+            </span>
+          </div>
+        </div>
+        <div class="row photo-container">
+          <div class="col-md-12">
+            <img class="profile-photo" src='assets/harry_potter_small.jpg'>
+            <h3 class="profile-username">Harry Potter</h3>
+          </div>
+        </div>
+      </div>
+
+      <!-- Timeline title bar -->
+      <div class="container-fluid">
+        <div class="row">
+          <div class="col-md-12">
+            <p class='about-bar'>Friends
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <!-- Section of friends -->
+      <!-- Main stuff here -->
+      <div class="container-fluid outlined information-panel are-you-code-reviewing">
+
+        <!-- Div class for the all encompassing row, for padding -->
+        <!-- All encompassing row -->
+        <div class="row">
+          <!-- Two columns for friends -->
+          <div class="col-md-6">
+            <!-- 3 rows of friends -->
+            <!-- First friend, column 1 -->
+            <div class="border-for-friends"> <!-- Applies border and spacing -->
+            <div class="row">
+              <div class="col-md-4">
+                <img src="assets/user_silhouette_generic.png" height="50px" width="50px">
+              </div>
+              <div class="col-md-4">
+                <a href="http://www.google.com/">Hermione Granger</a>
+                <p>432 Friends</p>
+              </div>
+              <div class="col-md-4">
+                <button class="btn btn-default bump-right">Unfriend Me</button>
+              </div>
+            </div>
+            </div> <!-- Border for friends -->
+
+            <!-- Second friend, column 1 -->
+            <div class="border-for-friends">
+            <div class="row">
+              <div class="col-md-4">
+                <img src="assets/user_silhouette_generic.png" height="50px" width="50px">
+              </div>
+              <div class="col-md-4">
+                <a href="http://www.google.com/">Hermione Granger</a>
+                <p>432 Friends</p>
+              </div>
+              <div class="col-md-4">
+                <button class="btn btn-default bump-right">Unfriend Me</button>
+              </div>
+            </div>
+            </div> <!-- Border for friends -->
+
+            <!-- Third friend, column 1 -->
+            <div class="border-for-friends">
+            <div class="row">
+              <div class="col-md-4">
```

Comment:
Why double-nest these two divs? You can apply two the "border-for-friends" class to the row as well.